packages/astro/src/index.ts

```diff
@@ -74,27 +62,11 @@ const createPlugin = (options?: SpotlightAstroIntegrationOptions): AstroIntegrat
             addDevOverlayPlugin(pluginPath);
           }
         } else if (options?.__debugOptions) {
-          injectScript('page', buildClientInitSnippet({ importPath: PKG_NAME, ...options }));
-        }
-      },
-
-      'astro:server:start': async ({ logger }) => {
-        if (options?.sidecarUrl) {
-          logger.debug('Detected custom sidecar URL. Skipping default sidecar setup.');
-          // If users set a custom sidecar URL, we assume they started the sidecar manually outside of Astro.
-          // So we don't setup the default sidecar instance.
-          return;
+          injectScript('page', buildClientInit(options));
         }
-
-        // Importing this dynamically because for some reason, the top level import
-        // caused a dev server error because the sidecar code was bundled into the server
-        const { setupSidecar } = await import('@spotlightjs/sidecar');
-        setupSidecar({ logger });
       },
     },
   };
 };
 
 export default createPlugin;
-
-export * from '@spotlightjs/overlay';
```

Comment:
Did we check if removing this export actually actually removes exports form the pacakge? If so that would be a breaking change and we shouldn't do it in a minor.

Reply:
I think it just allows people to use `@spotlightjs/astro` as if it was `@spotlightjs/spotlight` or `@spotlightjs/overlay` but with added functionality. So I'd not remove it.